composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerScreen.kt

```diff
@@ -53,11 +55,24 @@ fun TimerScreen(viewModel: TimerViewModel) {
         Box(
             modifier = Modifier.fillMaxSize().background(color = Color(0xFFFFFFFF)).padding(16.dp)
         ) {
-            Column(Modifier.align(Alignment.TopStart)) {
-                Workouts(list){ item -> viewModel.onDelete(item) }
+            Column(Modifier.align(Alignment.TopStart).fillMaxWidth().padding(top = 32.dp)) {
+                Row() {
+                    Workouts(list) { index -> viewModel.onClickedDelete(index) }
+                    Spacer(Modifier.weight(1f))
+                    RepsButton(
+                            reps,
+                        { viewModel.onEvent(TimerViewEvent.AddReps) },
+                         { viewModel.onEvent(TimerViewEvent.RemoveReps) }
+                    )
+
+                }
                 AddButton {
-                    viewModel.onEvent(TimerViewEvent.ClickAdd)
+                    viewModel.onEvent(TimerViewEvent.ClickedAdd)
                 }
```

Comment:
This `AddButton` should be a part of `Workouts`. The reason why this is a little odd is that you have these two things that are separate, and you are trying to orchestrate them together by this shared hard-coded width of `200.dp`.